packages/govuk-frontend-review/src/app.mjs

```diff
@@ -77,6 +77,9 @@ export default async () => {
   // Configure nunjucks
   const env = nunjucks.renderer(app)
 
+  // Used in the feature-flags middleware to set the govukRebrand global
+  app.set('nunjucksEnvGlobal', env)
```

Comment:
Hopefully the name/function of this property is pretty clear and it won't be an issue that it's globally available.

I guess if we wanted to lock it down a bit more, we could wrap the middleware in a function that takes the `env` as a param, then pop it into `res.locals` inside the middleware. Something like: `app.use(featureFlags(env));`

Or we could wrap the env in a function that only allows access to that env within the feature flags middleware.

That all seems a bit overkill to me personally?